eng/pipelines/coreclr/templates/crossgen2-comparison-job.yml

```diff
@@ -146,34 +148,39 @@ jobs:
             echo Targeting $(targetFlavor) ;
             chmod +x     $HELIX_WORKITEM_PAYLOAD/corerun;
             mkdir -p     $HELIX_WORKITEM_PAYLOAD/log;
-            export CORE_ROOT=$HELIX_CORRELATION_PAYLOAD;
+            export CORE_ROOT=$HELIX_WORKITEM_PAYLOAD;
             python3 -u $HELIX_CORRELATION_PAYLOAD/crossgen2_comparison.py crossgen_framework
             --crossgen    $HELIX_WORKITEM_PAYLOAD/crossgen2/crossgen2.dll
             --dotnet      $HELIX_WORKITEM_PAYLOAD/corerun
             --core_root   $HELIX_WORKITEM_PAYLOAD/prebuiltWork/dlls
             --result_dir  $HELIX_WORKITEM_PAYLOAD/log
             --target_os   $(target_crossgen2_os)
-            --target_arch $(targetarch);
+            --target_arch $(targetarch)
+            --compiler_arch_os $(compiler_arch_os);
```

Comment:
Is the semicolon appropriate here?

Reply:
Yes. Its the separator between different commands in bash. Just like the '&'  for cmd scripts below.